### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by thelovekesh**
Fixed some common typing errors and make readme.me google translate compatible so that anyone from another language can understand it by translating.

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,14 +147,14 @@ Onboard your workload onto the service mesh with confidence. Check your service
...
 <!-- <a href="https://raw.githubusercontent.com/layer5io/meshery/master/docs/assets/img/readme/Meshery-Grafana-Charts.png"><img alt="Meshery Grafana Boards" src="docs/assets/img/readme/Meshery-Grafana-Charts.png" style="padding-top:10px;margin-left:10px;" width="45%" align="right" /></a> -->
 <h4>Standardized Service Performance Management</h4>
 
-Meshery is the service-mesh-neutral utility for uniformly managing the performance of services and the meshes that run them. As an implementation of the Service Mesh Performance ([SMP](https://layer5.io/performance)), Meshery enables you to measure the value provided by a service mesh in context of the overhead incurred.
+Meshery is the service-mesh-neutral utility for uniformly managing the performance of services and the meshes that run them. As an implementation of the Service Mesh Performance ([SMP](https://layer5.io/performance)), Meshery enables you to measure the value provided by a service mesh in the context of the overhead incurred.
```
Comment (leecalcote) [inline README.md:157]:
```suggestion
Meshery is the service-mesh-neutral utility for uniformly managing the performance of services and the meshes that run them. As an implementation of the Service Mesh Performance ([SMP](https://layer5.io/performance)), Meshery enables you to measure the value provided by a service mesh in context of the overhead incurred.
```

File: README.md [hunk 2]
```diff
@@ -147,14 +147,14 @@ Onboard your workload onto the service mesh with confidence. Check your service
 
 <h3>Adhering to Service Mesh Standards</h3>
 
-In an effort to produce service mesh agnostic tooling, Meshery uses the [service mesh performance](https://smp-spec.io/) as a common format to capture and measure your mesh's performance against a universal service mesh performance index. As a partner of VMware's Multi-Vendor Service Mesh Interoperation (Hamlet) and Service Mesh Interface (SMI), Meshery participates in advancing service mesh adoption through standardization of APIs.
+In an effort to produce service mesh agnostic tooling, Meshery uses the [service mesh performance](https://smp-spec.io/) as a common format to capture and measure your mesh's performance against a universal service mesh performance index. As a partner of VMware's Multi-Vendor Service Mesh Interoperation (Hamlet) and Service Mesh Interface (SMI), Meshery participates in advancing service mesh adoption through the standardization of APIs.
```
Comment (leecalcote) [inline README.md:150]:
```suggestion
In an effort to produce service mesh agnostic tooling, Meshery uses the [service mesh performance](https://smp-spec.io/) as a common format to capture and measure your mesh's performance against a universal service mesh performance index. As a partner of VMware's Multi-Vendor Service Mesh Interoperation (Hamlet) and Service Mesh Interface (SMI), Meshery participates in advancing service mesh adoption through standardization of APIs.
```

File: README.md [hunk 3]
```diff
@@ -129,7 +129,7 @@ See the [getting started](https://meshery.io/#getting-started) section to quickl
 <h3>Service Mesh Lifecycle Management</h3>
 Meshery manages the provisioning, configuration and operation your service mesh. While supporting different types of service meshes, Meshery also offers a simple way to explore each service mesh and compare them using bundled sample applications.
 
-Interoperate multiple service meshes with service mesh adapters provision, configure, and manage their respective service meshes. Meshery is an implementation of Service Mesh Interface (SMI).
+Interoperate multiple service meshes with service mesh adapters provision, configure, and manage their respective service meshes. Meshery is an implementation of the Service Mesh Interface (SMI).
```
Comment (leecalcote) [inline README.md:132]:
```suggestion
Interoperate multiple service meshes with service mesh adapters provision, configure, and manage their respective service meshes. Meshery is an implementation of Service Mesh Interface (SMI).
```

File: README.md [hunk 4]
```diff
@@ -169,7 +169,7 @@ Meshery provides tooling to validate any service mesh that claims to implement a
 ✔︎ Produces compatibility matrix. <br />
 ✔︎ Ensures provenance of results. <br />
 ✔︎ Runs a set of conformance tests. <br />
-✔︎ Built into participating service mesh’s release pipeline. <br />
+✔︎ Built into the participating service mesh’s release pipeline. <br />
```
Comment (leecalcote) [inline README.md:172]:
```suggestion
✔︎ Built into participating service mesh’s release pipeline. <br />
```

File: README.md [hunk 5]
```diff
@@ -221,7 +221,7 @@ Find out more on the <a href="https://layer5.io/community#meshmate">Layer5 commu
 
 ## Contributing (yes!)
 
-We're a warm and welcoming community of open source contributors. Please join. All types of contribution are welcome. Be sure to read the <a href="https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit">Meshery Contributors Welcome Guide</a> for a tour of resources available to you and how to get started.
+We're a warm and welcoming community of open source contributors. Please join. All types of contributions are welcome. Be sure to read the <a href="https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit">Meshery Contributors Welcome Guide</a> for a tour of resources available to you and how to get started.
```
Comment (leecalcote) [inline README.md:224]:
```suggestion
We're a warm and welcoming community of open source contributors. Please join. All types of contribution are welcome. Be sure to read the <a href="https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit">Meshery Contributors Welcome Guide</a> for a tour of resources available to you and how to get started.
```

File: README.md [hunk 6]
```diff
@@ -196,7 +196,7 @@ Our projects are community-built and welcome collaboration. 👍 Be sure to see
 <a href ="https://layer5.io/community"><img alt="MeshMates" src="docs/assets/images/Layer5-MeshMentors.png" style="margin-right:10px; margin-bottom:7px;" width="28%" align="left" /></a>
 <h3>Find your MeshMate</h3>
 
-<p>MeshMates are experienced Layer5 community members, who will help you learn your way around, discover live projects and expand your community network.
+<p>MeshMates are experienced Layer5 community members, who will help you learn your way around, discover live projects, and expand your community network.
```
Comment (leecalcote) [inline README.md:199]:
```suggestion
<p>MeshMates are experienced Layer5 community members, who will help introduce you to Layer5 projects, others in the community, and generally assist you helping you succeed as an open source contributor.
```

File: README.md [hunk 7]
```diff
@@ -252,7 +252,7 @@ We're a warm and welcoming community of open source contributors. Please join. A
 
 ### License
 
-This repository and site are available as open source under the terms of the [Apache 2.0 License](https://opensource.org/licenses/Apache-2.0).
+This repository and site are available as open-source under the terms of the [Apache 2.0 License](https://opensource.org/licenses/Apache-2.0).
```
Comment (leecalcote) [inline README.md:255]:
```suggestion
This repository and site are available as open source under the terms of the [Apache 2.0 License](https://opensource.org/licenses/Apache-2.0).
```

File: README.md [hunk 8]
```diff
@@ -262,4 +262,4 @@ This repository and site are available as open source under the terms of the [Ap
 
 **Open Source First**
 
-<p>Our projects establish industry standards and enable service developers, owners, and operators with repeatable patterns and best practices for managing all aspects of distributed services. Our shared commitment to the open source spirit push the Layer5 community and its projects forward.</p>
+<p>Our projects establish industry standards and enable service developers, owners, and operators with repeatable patterns and best practices for managing all aspects of distributed services. Our shared commitment to the open-source spirit push the Layer5 community and its projects forward.</p>
```
Comment (leecalcote) [inline README.md:265]:
```suggestion
<p>Our projects establish industry standards and enable service developers, owners, and operators with repeatable patterns and best practices for managing all aspects of distributed services. Our shared commitment to the open source spirit push the Layer5 community and its projects forward.</p>
```

**Review by leecalcote — COMMENTED** [review]
Thanks for jumping in, @thelovekesh.

I tweaked or reverted a few items. Overall, this is an improvement. Much appreciated, @thelovekesh.

**Review by leecalcote — CHANGES_REQUESTED** [review]
@thelovekesh will you sign-off on your commit by executing the git commands outlined here- https://github.com/layer5io/meshery/pull/1641/checks?check_run_id=1204950094?

All Layer5 projects require a DCO (sign-off on commits). Thanks!

**Comment by thelovekesh** [comment]
Done.:)

**Review by leecalcote — APPROVED** [review]
Excellent. Thank you, @thelovekesh. 👍

**Comment by leecalcote** [comment]
🎈 Yay, @thelovekesh!

**Pull request merged**